### PR TITLE
Learning Hub launch

### PR DESCRIPTION
**Pull request by davidfischer**
- Add background image
- Add footer link to the hub

I figure we can add a header link once the "About Surveillance Advertising" section is done. Thoughts?

### CODE REVIEW

**Review by ericholscher — APPROVED** [review]
I think a header link is fine. It's definitely more useful in the header already than the vision, so I don't think we need all the content to link it 👍

File: ethicalads-theme/templates/includes/footer.html [hunk 1]
```diff
@@ -134,7 +134,7 @@ <h6 class="font-weight-bold text-uppercase text-gray-700">Company</h6>
             <a href="/blog/" class="text-reset">Blog</a>
           </li>
           <li class="mb-3">
-            <a href="/advertising-vision/" class="text-reset">Ad Vision</a>
```
Comment (ericholscher) [inline ethicalads-theme/templates/includes/footer.html:137]: I wonder if we should link our vision a couple more places if we aren't already, given that we're moving it out of the header. Might be useful in 1 or 2 places, or maybe keep it in the footer?

**Comment by davidfischer** [comment]
I put the header link as "Learning Hub" but I almost think something like "Understand Ethical Ads" or something more descriptive might be better. Thoughts?

**Comment by MxRaCohen** [comment]
> I put the header link as "Learning Hub" but I almost think something like "Understand Ethical Ads" or something more descriptive might be better. Thoughts?

I don't like "Understand Ethical Ads". I could see "Resource Center" or maybe "Discover Ethical Ads"

**Comment by davidfischer** [comment]
I think "resource center" is about the same as "learning hub". "Discover EthicalAds" isn't bad though. I sort of like that.

**Pull request merged**